### PR TITLE
Fixes #11. Add basic express routing for needsdiagnosis dashboard

### LINKED ISSUE #11: Build simple node app w/ routing
To start with, we'll have a single route `/needsdiagnosis`, which will serve our first dashboard. cc @magsout.

I'm gonna build this with express (and use that to power the JSON API we need).

### PR DESCRIPTION
**Pull request by miketaylr**
r? @magsout 

We obviously don't need to use the .hbs stuff, but the express generator wants you to choose a templating engine.

(I found this article on how to integrate express + react dev server, in case that's useful:

https://daveceddia.com/create-react-app-express-backend/)

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -14,13 +14,13 @@
     "curly": [2, "all"],
     "eol-last": "error",
     "eqeqeq": [2, "smart"],
-    "indent": [2, 2],
```
Comment (miketaylr) [inline .eslintrc]: prettier will take care of this for us.

File: routes/users.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+var express = require("express");
```
Comment (miketaylr) [inline routes/users.js]: Eh, I guess we don't need this.

**Comment by miketaylr** [comment]
No reason to not merge now, I think.

**Comment by magsout** [comment]
👍

**Pull request merged**